config/ignition_fortress_ubuntu_focal.yaml

```diff
@@ -118,9 +102,6 @@ filter_formula: "\
  Package (= libignition-sensors6-thermal-camera) |\
  Package (= libignition-sensors6-thermal-camera-dev) \
 ), $Version (% 6.3.0-2*) |\
-(Package (= ignition-tools) |\
- Package (= libignition-tools-dev) \
-), $Version (% 1.4.1-1*) |\
```

Comment:
I would prefer that these changes are revered for the reasons described in https://github.com/ros-infrastructure/reprepro-updater/pull/168#issuecomment-1218663623

Reply:
Reverted [adf0dce](https://github.com/ros-infrastructure/reprepro-updater/pull/168/commits/adf0dcee76545baaff671047521d91dad91e8176)